documentation/articles/xtoolkit/connectivity.md

```diff
@@ -16,6 +38,15 @@ Please, use [IConnectivityService](xref:Softeq.XToolkit.Connectivity.IConnectivi
 var isConnected = _connectivityService.IsConnected;
 ```
 
+**Advanced**: Strict check connectivity only for most common mobile connections:
```

Comment:
Can you please point out what is the exact problem that is fixed by this code? It's not really obvious when you should use a simple connectivity check and when you should use the solution you've provided here

Reply:
Deleted